server/server.go

```diff
@@ -194,33 +233,32 @@ func (s *RPCServer) consume(queueName string, handler handlerFunc, inputCh *amqp
 		return err
 	}
 
+	// attach the middlewares to the handler.
```

Comment:
```python
>>> s = "attach"
>>> s[0].upper() + s[1:]
```

Reply:
This goes for a lot of comments, don't know why I put it here. :)

Reply:
Yea, I'll give all the comments a once over.